pkg/sql/show_test.go

```diff
@@ -542,21 +542,19 @@ func TestShowJobs(t *testing.T) {
 		in.id, in.status, in.created, inPayload,
 	)
 
-	for _, source := range []string{"[SHOW JOBS]", "crdb_internal.jobs"} {
-		var out row
-		sqlDB.QueryRow(fmt.Sprintf(`
+	var out row
+	sqlDB.QueryRow(fmt.Sprintf(`
 			SELECT
 				id, type, status, created, description, started, finished, modified,
-				fraction_completed, username, descriptor_ids, error, coordinator_id
-			FROM %s`, source),
-		).Scan(
-			&out.id, &out.typ, &out.status, &out.created, &out.description, &out.started,
-			&out.finished, &out.modified, &out.fractionCompleted, &out.username, &out.descriptorIDs,
-			&out.err, &out.coordinatorID,
-		)
-		if !reflect.DeepEqual(in, out) {
-			diff := strings.Join(pretty.Diff(in, out), "\n")
-			t.Fatalf("in job did not match out job:\n%s", diff)
-		}
+				fraction_completed, username, error, coordinator_id
+			FROM crdb_internal.jobs`, source),
+	).Scan(
+		&out.id, &out.typ, &out.status, &out.created, &out.description, &out.started,
+		&out.finished, &out.modified, &out.fractionCompleted, &out.username, &out.descriptorIDs,
+		&out.err, &out.coordinatorID,
+	)
+	if !reflect.DeepEqual(in, out) {
+		diff := strings.Join(pretty.Diff(in, out), "\n")
+		t.Fatalf("in job did not match out job:\n%s", diff)
```

Comment:
It makes me somewhat nervous to stop testing `SHOW JOBS`, but I don't have a great alternative.